auth/server/src/main/java/com/seagates3/authentication/ClientRequestParser.java

```diff
@@ -94,7 +94,8 @@ public static ClientRequestToken parse(FullHttpRequest httpRequest,
 
         if (authorizationHeader == null ||
             authorizationHeader.replaceAll("\\s+", "").isEmpty()) {
-            return null;
+          ServerResponse serverResponse = responseGenerator.AccessDenied();
```

Comment:
looks fine, see it doesn't break other ceph/splunk tests

Reply:
yes. will run entire splunk suite to check if it cause any regression or not

Reply:
ran splunk and ceph tests and no new regression with this change